solutions/instance/README.md

```diff
@@ -0,0 +1,11 @@
+# Cloud Object Storage for IBM Cloud - Financial Services Cloud solution
+
+This architecture creates an Cloud Object Storage and bucket on IBM Cloud® and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- A Cloud Object Storage Instance for IBM Cloud
```

Comment:
`A Cloud Object Storage Instance for IBM Cloud` -> A Cloud Object Storage instance on IBM Cloud